test/functional/auth/test_password_policy.py

```diff
@@ -13,17 +13,21 @@
 # limitations under the License.
 #
 
+from pprint import pprint as pp
+
 
 class TestPasswordPolicy(object):
     def test_load(self, bigip):
-        pp = bigip.auth.password_policy.load()
-        assert pp.maxLoginFailures == 0
-        pp.refresh()
-        assert pp.maxLoginFailures == 0
+        password_policy = bigip.auth.password_policy.load()
```

Comment:
Should this test be more inline with what @tar saw when we loaded an object twice?